packages/playground/src/views/dashboardsuite_view.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <view-layout>
+    <tf-casperlabs />
+
+    <template #list>
+      <TfDeploymentList title="Dashboard Suite Instances" :project-name="name" />
+    </template>
+  </view-layout>
+</template>
+
+<script lang="ts">
+import { ProjectName } from "../types";
+import TfCasperlabs from "../weblets/tf_dashboardsuite.vue";
+import TfDeploymentList from "../weblets/tf_deployment_list.vue";
+export default {
+  name: "DashboardSuiteView",
+  components: {
+    TfDeploymentList,
+    TfCasperlabs,
```

Comment:
Why?

Reply:
Why TfCasperlabs? Because it worked with this but didn't work with TfDashboardSuite. I couldn't find how to make it work. If you have some potential fixes I could work with, it would be appreciated. Thanks.